contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -124,16 +137,16 @@ impl Contract {
         } else {
             (None, None)
         };
-
         self.available_players.insert(account_id,
             &GameConfig {
                 token_id: AccountId::new_unchecked("near".into()),
                 deposit,
                 opponent_id,
                 referrer_id: referrer_id.clone(),
-                created_at: cur_timestamp
+                created_at: nano_to_sec(cur_timestamp),
```

Comment:
maybe we can include `from` and `to` in this structure, and remove `created_at` and remove the `self.player_availability` container.

Reply:
Sure we can do it. I did it in separate structure since you suggested this will be needed for the indexer in the future. Let me know

Reply:
let's have only one struct. Indexer will easily handle this.